src/gsi/read_dbz_nc.f90

```diff
@@ -481,6 +483,9 @@ subroutine read_dbz_nc(nread,ndata,nodata,infile,lunout,obstype,sis,hgtl_full,no
         !!end modified for thinning
      
          thisazimuthr=0.0_r_kind
+         thistiltr=0.0_r_kind
+         this_stahgt=0.0_r_kind
+         thisrange=0.0_r_kind
```

Comment:
To prevent undefined values detected in the case of "DIAG_RADARDBZ=T", I just added this part. After this modification, undefined values was no longer detected in the test.

Reply:
@shoyokota Thanks for tracing them out.  Those values are not (should not be) used in the following analysis.  To avoid future mistaken use of them , I'd recommend to use missing value (r_missing) rather than 0 to define them.

Reply:
@TingLei-NOAA Thanks. I modified here to use r_missing, and I confirmed that the analysis was not changed.